README.md

```diff
@@ -5,19 +5,21 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/friendsofcake/bootstrap-ui.svg?style=flat-square)](https://packagist.org/packages/friendsofcake/bootstrap-ui)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://packagist.org/packages/friendsofcake/bootstrap-ui)
 
-Transparently use [Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
+Transparently use [Bootstrap 4][twbs4] with [CakePHP 3][cakephp].
 
 ## Requirements
 
 * CakePHP 3.x
-* Bootstrap 3.x
-* jQuery 1.9+
+* Bootstrap 4.x
+* npm 5.x
+* jQuery 3.2+
+* Popper.js 1.x
 
 ## What's included?
 
-- FlashComponent + elements (types: error, info, success, warning)
-- FormHelper (align: default, inline, horizontal)
-- HtmlHelper (components: breadcrumbs, badge, label, icon)
+- FlashComponent + elements (types: primary, secondary, success, danger, warning, info, light, dark)
```

Comment:
Can we keep an error and info alias? And just mark as deprecated. Will ease the transition.

Reply:
How do the controller calls look like?
I think we should actually keep the 4 types we always had (error, info, success, warning), and instead allow the others to be opt-in on top if desired.
But error in cakephp is just internally transferred into a style class called "danger". But leaking those classes into the top layer is not good IMO.

Reply:
Yes, `error`, `info`, `success`, `warning` should be the the only valid types. Just the FlashHelper should set associated class name based on these 4 types.